src/services/resource-getter.js

```diff
@@ -2,11 +2,22 @@ const _ = require('lodash');
 const Interface = require('forest-express');
 const createError = require('http-errors');
 const utils = require('../utils/schema');
+const ResourcesGetter = require('./resources-getter');
 
 class ResourceGetter {
-  constructor(model, params) {
+  constructor(model, params, user) {
     this._model = model;
     this._params = params;
+    this._user = user;
+  }
+
+  async _isAllowed() {
+    const params = {
+      timezone: this._params.timezone,
+      filters: JSON.stringify({ field: '_id', operator: 'equal', value: this._params.recordId }),
+    };
+
+    return await new ResourcesGetter(this._model, null, params, this._user).count() === 1;
```

Comment:
[MISC] I don't understand why this is needed 🤔

Reply:
If I understand well, you perform a raw `ResourcesGetter.perform` to know if the record exist before actually performing a get query?

Reply:
I perform a count() to know is the record is _allowed_
When ResourcesGetter counts, it will apply the filters, and make the $lookUps which are necessary